src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -369,7 +275,7 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
 
 
     @Override
-    void queryJobAbortion(List<BEJob> executedJobs) {
+    void killJobs(List<BEJob> executedJobs) {
```

Comment:
Please check, if this can somehow be unified with the code from the other managers.

Reply:
Done